IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -377,6 +377,22 @@ - (void)decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction
         return;
     }
     
+    // Handle anchor links that were clicked
+    if([navigationAction navigationType] == WKNavigationTypeLinkActivated) {
+#if TARGET_OS_IPHONE
+        UIApplication *application = [UIApplication performSelector:@selector(sharedApplication)];
+#else
+        NSWorkspace *application = [NSWorkspace sharedWorkspace];
+#endif
+        //Open secure web links with target=new window in default browser
+        if(([requestURL.scheme.lowercaseString isEqualToString:@"https"] && !navigationAction.targetFrame.isMainFrame) ||
+           (![requestURL.scheme.lowercaseString hasPrefix:@"http"])) {  // Open non-web links with URL schemes that can be opened by the application
+            [application performSelector:@selector(openURL:) withObject:requestURL];
```

Comment:
Does this work on NSWorkspace as well? I'd prefer to have a utility that knows how to open URL depending on the OS. I also see we have similar code in MSIDWebOpenBrowserResponseOperation.m

Reply:
Since embedded webview is also available for macOS targets, I think this should handle links in mac apps as well.

In context of MFA setup, server doesn't show activate MFA link and just shows the QR code. The user will have to download Auth app on their device and scan the QR code.